frontend/storybook/src/components/entry/imagelabel/one.ts

```diff
@@ -0,0 +1,38 @@
+import "@elements/entry/admin/images/meta/page";
```

Comment:
on the file itself - there is no `imagelabel` entry, should be in `admin/images/meta/`